server/server.js

```diff
@@ -252,6 +254,17 @@ app.get('/dashboard/:installationId?', async (req, res) => {
   }
 })
 
+app.get('/buttons-data', sessionChecker, async (req, res) => {
+  const data = await db.getDataForExport()
+  const csv = csvParser.parse(data)
+
+  const millis = Date.now()
+  const timestamp = new Date(millis).toISOString().slice(0, -5).replace(/T|:/g, '_')
+  console.log(data)
```

Comment:
I think this `console.log` should be removed.

Reply:
Gone!